e2e/__tests__/jestChangedFiles.test.ts

```diff
@@ -20,18 +19,11 @@ const DIR = path.resolve(tmpdir(), 'jest-changed-files-test-dir');
 
 const GIT = 'git -c user.name=jest_test -c user.email=jest_test@test.com';
 const HG = 'hg --config ui.username=jest_test';
+const testIfHg = buildTestIfHg();
```

Comment:
There's no `build` with the current approach

Reply:
Arf forgot this one, I'll push another patch. Sorry I just applied the suggestion. I'll fix it and re-run everything on my side